### PR TITLE
Update admin guide for ephemeral storage

### PR DESCRIPTION
**Pull request by ghost**
This is a follow-up to https://github.com/openshift/openshift-docs/pull/10828.

@aheslin 

http://file.rdu.redhat.com/tmorriso/071618/ephemeral-storage-docs/admin_guide/allocating_node_resources.html

http://file.rdu.redhat.com/tmorriso/071618/ephemeral-storage-docs/admin_guide/limits.html

http://file.rdu.redhat.com/tmorriso/071618/ephemeral-storage-docs/admin_guide/overcommit.html#overcommit-ephemeral-storage

http://file.rdu.redhat.com/tmorriso/071618/ephemeral-storage-docs/admin_guide/quota.html

### CODE REVIEW

**Comment by tmorriso-rh** [comment]
@jhou1 @liangxia Please provide QE review.

**Comment by tmorriso-rh** [comment]
@openshift/team-documentation PTAL.

**Comment by tmorriso-rh** [comment]
@kalexand-rh Do you mind doing the peer review for this PR since it's related to the others you reviewed? Thanks!

**Comment by kalexand-rh** [comment]
@tmorriso-rh, I will review this one and the other one you have open. I needed to switch gears for a bit.

File: admin_guide/limits.adoc [hunk 1]
```diff
@@ -28,10 +28,10 @@ default value is applied to the resource.
 
 ifdef::openshift-origin,openshift-enterprise[]
 NOTE: As of {product-title} 3.10, it is possible to specify limits and
```
Comment (kalexand-rh) [inline admin_guide/limits.adoc]: s/it is possible to/you can

**Comment by tmorriso-rh** [comment]
@kalexand-rh Thanks for reviewing all of these PRs! I am reviewing/incorporating your suggestions today.

**Comment by kalexand-rh** [comment]
@tmorriso-rh, sounds good! Let me know if there's anything that's not clear or that you want to discuss.

**Comment by tmorriso-rh** [comment]
@vikram-redhat Can you contact me about this PR? Thanks.

**Pull request merged**